### ISSUE TITLE
training dinov2 with 1 node

### ISSUE THREAD

**Issue by shashankvkt** [facebookresearch/dinov2#507]
Hello,
Thanks a lot for sharing your amazing work. I was wondering if you can please share the config files for base and large models  of dinov2 for training it with just 1 node (4 gpus). I had tried running the code with 1 gpu using batch_size_per_gpu 256 on 1 H100 gpu and the loss doesnt converge. 

I assume the hyper-parameters have been optimized for multi-node training, but i was wondering if you can share any pointers to make it work for 1 node.

Thanks
